packages/gafl-webapp-service/src/session-cache/__tests__/session-manager.spec.js

```diff
@@ -82,4 +100,15 @@ describe('The user', () => {
     expect(response.statusCode).toBe(302)
     expect(response.headers.location).toBe(CONTROLLER.uri)
   })
+
+  it('returns 200 when attempting to access ' + IDENTIFY.uri, async () => {
```

Comment:
You can use template quotes here rather than concatenation: `` `returns 200 when attempting to access ${IDENTIFY.url}` ``